multi_label_trainer_tfidf.ipynb

```diff
@@ -63,9 +63,10 @@
         "from sklearn.model_selection import train_test_split\n",
```

Comment:
<p>Before this, it might be a good idea to add one/two lines on the training progress plot. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>